components/playground.tsx

```diff
@@ -336,48 +549,52 @@ export default function Playground({
 							</FormItem>
 						)}
 					/>
-					{userId && guildMember !== undefined ? (
-						<FormField
-							control={form.control}
-							name="sendToDb"
-							render={({ field }) => (
-								<FormItem>
-									<FormLabel>Register this passport</FormLabel>
-									<FormDescription>
-										{registerCheckboxDisabled
-											? "Sorry, we're at capacity for this upcoming passport ceremony :("
-											: "Sign up for the passport ceremony this Friday at 10pm at Hack Night. By checking this box you pinky promise you will ACTUALLY be there this Friday. If you can't make it please DM Matthew so we don't prep for you!"}
-									</FormDescription>
-									<FormControl>
-										<Checkbox
-											className="h-6 w-6"
-											disabled={registerCheckboxDisabled}
-											onCheckedChange={(e) => {
-												// it hasn't updated the value at this point yet, so it's the opposite actually
-												if (field.value === true) {
-													setGenerationSteps(GENERATION_STEPS.base);
-												} else {
-													setGenerationSteps(GENERATION_STEPS.register);
-												}
-												return field.onChange(e);
-											}}
-											checked={field.value}
-										/>
-									</FormControl>
-									<FormMessage />
-								</FormItem>
-							)}
-						/>
-					) : null}
-					<Button className="amberButton" type="submit" disabled={isLoading}>
-						{isLoading ? "Generating..." : "Generate"}
-					</Button>
+					{form.getValues("sendToDb") == "true" ? (
+						<Dialog>
+							<DialogTrigger
+								className="rounded-[0.25rem] border-2 border-slate-800 bg-amber-500 px-4 py-2 text-sm font-bold text-slate-800 shadow-[4px_4px_0_0_#0f172a] transition-colors hover:bg-amber-500/90 focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-white focus-visible:ring-offset-0"
+								disabled={isLoading}
+							>
+								{isLoading ? "Generating..." : "Generate"}
+							</DialogTrigger>
+							<DialogContent className="w-11/12 sm:w-auto max-h-screen h-128 overflow-y-auto">
+								<DialogHeader>
+									<DialogTitle className="text-2xl">
+										Passport Ceremony Registration
+									</DialogTitle>
+								</DialogHeader>
+								<DialogDescription className="flex flex-col gap-2 leading-relaxed text-base">
+									<p>PLACEHOLDER TEXT HERE</p>
+									<p>PLEASE MAKE SURE YOU CAN MAKE IT TO THE CEREMONY</p>
+									<p>IF SOMETHING COMES UP TELL MATTHEW</p>
+									<DialogClose asChild>
+										<Button
```

Comment:
<img width="617" alt="Screenshot 2024-09-24 at 8 02 39 PM" src="https://github.com/user-attachments/assets/83614ebe-b6fb-41d4-82ae-f55e26f75965">

I still think this looks weird. I suggest removing the padding between the heading and the text.

Also, let's change placeholder text to:

"By clicking "Register My Passport", you're signing up to make your own passport at the passport ceremony on <date> at <time>. Please show up on time so that you have enough time to make your passport. If you're late, we will start without you & you will need to re-register for the next one.

"We spend real money and human labor on each passport, so please make sure you're there! If plans change & you can't make it, please post in #lounge or DM @matthew and let us know so that we don't prep the materials for your passport."

Reply:
Resolved in [00d29d9](https://github.com/purduehackers/passport-issuing-office/pull/52/commits/00d29d972990892c27894b10c9ce4850b3c19ed8)